qsc/extract.py

```diff
@@ -18,22 +18,22 @@
 """Extracting archives and stuff"""
 
 import os
-import zipfile
+import tarfile
 
 import qsc
 
 def extract_release(release):
-    name = "qt-everywhere-src-{}".format(release)
+    name = "qt-everywhere-opensource-src-{}".format(release)
 
-    zip_path = os.path.join("archives", name+".zip")
+    tar_path = os.path.join("archives", name+".tar.xz")
     
     print("Extracting...", end="", flush=True)
     
     if qsc.USE_CACHE and os.path.isdir(name):
```

Comment:
When extracted the directory within the archive is still `qt-everywhere-src` not `qt-everywhere-opensource-src`.
```suggestion
    if qsc.USE_CACHE and os.path.isdir("qt-everywhere-src-{}".format(release)):
```